woocommerce/grow
# Reorganize the package directory structure for the GitHub custom actions

### Changes proposed in this Pull Request:

After adding the GitHub custom actions written in PHP, a similar directory structure was added to [packages/php/github-actions](https://github.com/woocommerce/grow/tree/9cef99cac2b4ad1e08ef12d8432928d59625d817/packages/php/github-actions). Ideally, the structure of GitHub custom actions would be better not to categorize by programming languages. To unify it, this PR:
- Move the `packages/js/github-actions` directory under the `packages` directory
- Migrate the `packages/php/github-actions` directory into the `packages/github-actions` directory
- Extra change: Fix syntax for looping PHP directories in build-php-actions.sh

Some relevant discussions:
- https://github.com/woocommerce/grow/pull/12#issuecomment-1152324218
- p1683171180419589-slack-CK365S85V

### Detailed test instructions:

1. View the test build job of this PR: https://github.com/woocommerce/grow/actions/runs/5317960808
2. Compare the [test build revision](https://github.com/woocommerce/grow/tree/actions-v1.9.0-pre) with [actions-v1.8.0](https://github.com/woocommerce/grow/tree/actions-v1.8.0)
   - All actions should be moved to the top directory
   - No files were missed to be committed
   - In particular, the PHP `hook-documentation` action was integrated into the original structure of JavaScript and composite actions in this PR and has a few more corresponding adjustments.
3. Check if each hyperlink in README files is linked to the correct page
   - [README.md](https://github.com/woocommerce/grow/blob/c383c518170ac5a7ecdc9f1e50b34bb6605aad6b/README.md)
   - [packages/github-actions/README.md](https://github.com/woocommerce/grow/blob/c383c518170ac5a7ecdc9f1e50b34bb6605aad6b/packages/github-actions/README.md)
